### PR TITLE
useradd, userdel, usermod: add Spanish translation

### PR DESCRIPTION
**Pull request by victorhck**
- [X] The page (if new), does not already exist in the repository.
- [X] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [X] The page has 8 or fewer examples.
- [X] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [X] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages.es/linux/userdel.md [hunk 1]
```diff
@@ -1,8 +1,21 @@
...
+> Nota: todos los comandos deben ser ejecutados como root.
 > Más información: <https://manned.org/userdel>.
 
-- Elimina un usuario y su directorio personal:
+- Elimina un usuario:
 
-`userdel -r {{nombre_de_usuario}}`
+`userdel {{nombre}}`
+
+- Elimina un usuario junto con su directorios home y mail spool:
```
Comment (navarroaxel) [inline pages.es/linux/userdel.md:11]:
```suggestion
- Elimina un usuario junto con su directorio home y mail spool:
```

File: pages.es/linux/usermod.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+`usermod -l {{nuevo_nombre}} {{usuario}}`
+
+- Añade un usuario a grupos suplementarios (tener en cuenta los espacios en blanco):
+
+`usermod -a -G {{grupo1,grupo2}} {{usuario}}`
+
+- Crea un nuevo directorio home para un usuario y mueve sus archivos a él:
+
+`usermod -m -d {{ruta/a/home}} {{usuario}}`
```
Comment (navarroaxel) [inline pages.es/linux/usermod.md:16]:
```suggestion
`usermod -m -d {{ruta/al/home}} {{usuario}}`
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]

**Comment by victorhck** [comment]
agree!

**Review by navarroaxel — APPROVED** [review]

**Review by marchersimon — APPROVED** [review]
Thanks @victorhck

**Comment by victorhck** [comment]
a pleasure! ;)

**Pull request merged**